packages/overlays/src/GlobalOverlayController.js

```diff
@@ -78,9 +82,24 @@ export class GlobalOverlayController {
    * Hides the overlay.
    */
   hide() {
+    if (this._expandedInvokerNode) {
+      this._expandedInvokerNode.setAttribute('aria-expanded', 'false');
+    }
     this._createOrUpdateOverlay(false, this._data);
   }
 
+  addInvokerNodes(nodes) {
+    nodes.forEach(node => {
+      node.addEventListener('click', () => {
+        node.setAttribute('aria-expanded', 'true');
+        this._expandedInvokerNode = node;
+        this.show(node);
```

Comment:
not really sure if it should return focus to this element always. Maybe we should add a guard that checks if elementToFocusAfterHide is also provided and if so, it wins?
This default behavior could stay then, and the check should be added on hide.